feature_layers/create-mobile-geodatabase/src/main/java/com/esri/samples/create_mobile_geodatabase/CreateMobileGeodatabaseController.java

```diff
@@ -234,18 +239,17 @@ private void closeGeodatabase() {
     // handle UI
     viewTableButton.setDisable(true);
     closeGeodatabaseButton.setDisable(true);
-    createGeodatabaseButton.setDisable(true);
-    label.setText("Restart sample to begin again");
+    createGeodatabaseButton.setDisable(false);
+    label.setText("Create mobile geodatabase");
```

Comment:
I think at this point we might be better just hiding this label since it will be clear from the UI what to click to begin again: then the "Click map to add features" prompt appears as a helpful hint.